### ISSUE TITLE
Testing a Dialog does not work

### ISSUE THREAD

**Issue by dalewking** [cashapp/paparazzi#619]
I would like to do a test of a simple composable that is a Dialog

**Steps to Reproduce**
Create a composable that is a dialog. For example:
```
@Composable
fun ProgressOverlay(text: String) {
    Dialog(
        onDismissRequest = { },
        DialogProperties(
            dismissOnBackPress = false,
            dismissOnClickOutside = false,
        ),
    ) {
        Box(
            contentAlignment = Alignment.Center,
            modifier = Modifier
                .background(Color.White, shape = RoundedCornerShape(8.dp))
                .testTag("progressOverlay"),

        ) {
            Column(
                Modifier.padding(20.dp),
                horizontalAlignment = Alignment.CenterHorizontally,
            ) {
                Text(
                    text,
                    fontSize = 18.sp,
                    modifier = Modifier
                        .testTag("progressOverlayMessage"),
                )
                Spacer(Modifier.height(20.dp))
                CircularProgressIndicator()
            }
        }
    }
}
```

Write a paprazzi test to display it: 
```
class ProgressOverlaySnapshot {
    @get:Rule
    val paparazzi = Paparazzi(
        deviceConfig = PIXEL_5,
        theme = "android:Theme.Material.Light.NoActionBar"
    )

    @Test
    fun exampleMessage() {
        paparazzi.snapshot {
            ProgressOverlay("This is a test")
        }
    }
}
```
**Expected behavior**
I expect the dialog to be renderedgets rendered in the snapshot, but it is not shown
![18725068ad0b201c404f959c7f2e0199f9f05b28](https://user-images.githubusercontent.com/346062/196584435-0d3d1c9e-07e5-4588-9389-7ac0adf36a73.png)

I note that Preview in android studio does render the dialog without issue

**Comment by dalewking** [comment]
I'm still on Chipmunk:

Android Studio Chipmunk | 2021.2.1 Patch 2
Build #AI-212.5712.43.2112.8815526, built on July 10, 2022

**Comment by jrodbx** [comment]
This is now fixed as of f34ab8b, presumably by 6df4752 or 0268445.

Repro: https://github.com/cashapp/paparazzi/compare/jrod/2022-12-15/issue_619?expand=1

**Comment by dalewking** [comment]
Getting back to trying to get Paparazzi added to our project and this seems to still be an issue. In this case I am on Flamingo and I can confirm Dialogs also do not show in the Compose preview either.
